### ISSUE TITLE
docker cleos get account error from v1.0.4:  Failed with error: Assert Exception (10) a.get_symbol() == b.get_symbol(): 

### ISSUE THREAD

**Issue by zhaowh** [EOSIO/eos#4171]
$ docker pull eosio/eos:20180617
$ docker run -d --name keosd -t eosio/eos:20180617 /opt/eosio/bin/keosd  --wallet-dir /opt/eosio/bin/data-dir --http-server-address=127.0.0.1:8900 
$ alias cleos='docker exec -i keosd /opt/eosio/bin/cleos --wallet-url http://127.0.0.1:8900 -u http://mainnet.eoscalgary.io '
$ cleos version client
Build version: c9b7a247
$ cleos get account eosnowbanker
permissions: 
     owner     1:    1 EOS7s8RDt3XyWv4cXJceHaaEhUvxJV35CW2UBTjPNfeEGu48FeDAC
        active     1:    1 EOS7s8RDt3XyWv4cXJceHaaEhUvxJV35CW2UBTjPNfeEGu48FeDAC
memory: 
     quota:       199 KiB    used:     195.3 KiB  

net bandwidth: 
     delegated:       0.0100 EOS           (total staked delegated to account from others)
     used:               343 bytes
     available:        4.128 KiB  
     limit:            4.463 KiB  

cpu bandwidth:
     delegated:       1.0000 EOS           (total staked delegated to account from others)
     used:             4.177 ms   
     available:        82.99 ms   
     limit:            87.17 ms   

EOS balances: 
     liquid:            0.0002 EOS
     staked:            0.0000 SYS
     unstaking:         0.0000 SYS
388699ms thread-0   main.cpp:2712                 main                 ] Failed with error: Assert Exception (10)
a.get_symbol() == b.get_symbol(): 

     total:

**Comment by cc32d9** [comment]
I see exactly the same behavior in mainnet:
```
root@ubuntu01:~# cleos -u https://mainnet.eoscanada.com get account cc32dninexxx
permissions: 
     owner     1:    1 EOS7txFiAr3fFzmQNUbxpnPV5ApYjq22igdYVYatFHgo1Vx6Vr553
        active     1:    1 EOS7txFiAr3fFzmQNUbxpnPV5ApYjq22igdYVYatFHgo1Vx6Vr553
memory: 
     quota:     7.954 KiB    used:     2.926 KiB  

net bandwidth: 
     delegated:       0.1000 EOS           (total staked delegated to account from others)
     used:               901 bytes
     available:        43.95 KiB  
     limit:            44.83 KiB  

cpu bandwidth:
     delegated:       0.1000 EOS           (total staked delegated to account from others)
     used:             4.989 ms   
     available:        3.767 ms   
     limit:            8.756 ms   

EOS balances: 
     liquid:            1.0000 EOS
     staked:            0.0000 SYS
     unstaking:         0.0000 SYS
2823887ms thread-0   main.cpp:2712                 main                 ] Failed with error: Assert Exception (10)
a.get_symbol() == b.get_symbol(): 

     total:     root@ubuntu01:~# 

```

**Comment by arhag** [comment]
Resolved by #4201.
